src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -76,13 +121,14 @@
             </div>
         </div>
     </div>
-    <div ref="thumbsWrapper"
-         class="c-imagery__thumbs-wrapper"
-         :class="{'is-paused': isPaused}"
-         @scroll="handleScroll"
+    <div
+        ref="thumbsWrapper"
+        class="c-imagery__thumbs-wrapper"
+        :class="{'is-paused': isPaused}"
+        @scroll="handleScroll"
     >
         <div v-for="(datum, index) in imageHistory"
-             :key="datum.url"
+             :key="datum[timeKey]"
```

Comment:
I was getting a duplicate key issue when switching between time modes, this change seemed to solve it. Somehow there were two datum with the same url in that situation. I can debug why THAT was happening if you like and switch back to the url?

Reply:
Yeah that sounds like it might actually be a bug, we should understand why that's happening. It sounds to me like maybe it's not clearing out the old data after it switches time systems.

Reply:
URL is guaranteed unique, otherwise there's an issue. Timestamp may not necessarily be unique in some time systems or missions (eg. if a bunch of images are received on the ground at once the Earth Received Time could be the same).

Reply:
Ok cool, I'll change back to url and see what's up with that error.